packages/entity-list/src/modules/list/sagas.js

```diff
@@ -72,18 +72,19 @@ export function* fetchEntitiesAndAddToStore(page) {
   if (!entityStore[page]) {
     const {sorting, limit, columnDefinition} = list
 
-    const {searchFilters} = input
     const formName = `${formBase}_list`
 
     const searchInputs = yield call(getSearchInputs)
+    // combine search filters from input and search form
+    searchInputs._filter = [...input.searchFilters || [], ...searchInputs._filter || []]
```

Comment:
searchInputs._filter = call(_union, input.searchFilters, searchInputs._filter)

would also work and is a bit easier to read in my opinion (comment is omittable)